packages/pn-personafisica-webapp/public/locales/it/recapiti.json

```diff
@@ -84,19 +84,19 @@
     "phone-new-code": "Non l’hai ricevuto? Controlla che il numero di cellulare sia corretto o",
     "new-code-link": "generane uno nuovo",
     "email-added": "Indirizzo e-mail",
-    "phone-added": "Numero di cellulare",
+    "sms-added": "Numero di cellulare",
     "valid-email": "Indirizzo e-mail non valido",
     "valid-phone": "Numero di cellulare non valido",
```

Comment:
is valid-phone or valid-sms?

Reply:
The validation use `courtesy-contacts.valid-phone`, should i replace with `valid-sms`?

Reply:
yes